src/components/dashboard/FeedItems/EventSettingsByType.js

```diff
@@ -45,6 +45,11 @@ const getEventSettingsByType = (theme, type) => {
       name: 'system-filled',
       withoutAmount: true,
     },
+    welcome: {
```

Comment:
seems like there is the same config that for invite and feedback. Perhaps they are the same event type or at least use the same config?

Reply:
The thing here is I need invite type to display different content on feed item inside list, and on Feed modal...so the type is different but they use same props from here...